modules/nw-rfhe-introduction.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-ptp.adoc
+:_content-type: CONCEPT
+[id="nw-rfhe-introduction_{context}"]
+= How Redfish hardware events work
+
+OpenShift Container Platform now provides a Baremetal Hardware Event proxy that enables applications running on bare-metal clusters to respond quickly to Redfish hardware changes and failures such as breaches of temperature thresholds, fan failure, disk loss, power outages, and memory failure. These hardware events are delivered over a reliable low-latency transport channel based on Advanced Message Queuing Protocol (AMQP). The latency of the messaging service is in the 10~20 millisecond range.
+
+The hardware event proxy provides a publish-subscribe service for the hardware events, where multiple applications can use REST APIs to subscribe and consume the events.
+The hardware event proxy supports hardware compliant with Redfish OpenAPI v1.8 or higher.
+
+[NOTE]
+====
+This feature is supported for single node OpenShift clusters only.
+====
+
+[id="rfhe-elements_{context}"]
+== Redfish hardware events data flow
+
+The following figure illustrates the basic Redfish hardware events data flow:
+
+.Redfish hardware events data flow
+image::211_OpenShift_Redfish_dataflow_0222.png[Redfish hardware events data flow]
+
+=== Redfish hardware
```

Comment:
My general comment is that this description list could be made a lot tighter with a judicious edit. You don't need to describe each and every item in the diagram. For example, the detail in `Redfish hardware`, `Redfish event`, `REST APIs`, `Application pod (Subscriber)`, `vDU`, `Application` list items are covered in the preceding paragraphs.

Reply:
Removed redundant content.